docs/tfuser/readme.md

```diff
@@ -15,113 +15,175 @@ It has 3 main commands:
-            }
+            "subnet": "172.20.1.0/24",
+            "wg_public_key": "ZdSn/3gDfD1urtDCOPdOFwF0XsTAidJTSpwPETegZGw=",
+            "allowed_ips": [
+              "172.20.1.0/24",
+              "100.64.20.1/32",
+              "172.20.2.0/24",
+              "100.64.20.2/32"
+            ],
+            "endpoint": "[2a02:1802:5e:0:1000::f6]:12345"
           },
           {
-            "type": "wireguard",
-            "prefix": "2a02:1802:5e:153f::/64",
-            "Connection": {
-              "ip": "<nil>",
-              "port": 5439,
-              "key": "UOqDqmhynYYRmn0E1tMP+176ApHH33tyZTO7Br49LBg=",
-              "private_key": "b0f3fb4d599e41a459c011ea2057348501e7a22f6be7a21d5e82cb3ce439ee57c2a231ffcc62cdc71eb42e8336a4b404146fa28b124d02c19216271cead7e3a5d096b730eb5190fe8a655340a78fa403053f22a4dd0548f79da777f4"
-            }
+            "subnet": "172.20.4.0/24",
+            "wg_public_key": "0VcC6PCOW4JDZERtOUljSKXiPfd0di5qRQ/SHFn9XCs=",
+            "allowed_ips": [
+              "172.20.4.0/24",
+              "100.64.20.4/32"
+            ],
+            "endpoint": "[2a02:1802:5e:0:225:90ff:fef4:40af]:12348"
           }
-        ],
-        "exit_point": 0
-      }
-    ],
-    "prefix_zero": "2a02:1802:5e::/64",
-    "exit_point": {
-      "ipv4_conf": null,
-      "ipv4_dnat": null,
-      "ipv6_conf": {
-        "addr": "fe80::1000:1/64",
-        "gateway": "fe80::1",
-        "metric": 0,
-        "iface": "public"
+        ]
       },
-      "ipv6_allow": []
-    },
-    "allocation_nr": 0,
-    "version": 1
+      {
+        "node_id": "BpTAry1Na2s1J8RAHNyDsbvaBSM3FjR4gMXEKga3UPbs",
+        "subnet": "172.20.4.0/24",
+        "wg_private_key": "4ca059feb0831ed84dfa5a436fcbfa3a23e9b57766b1a1a2d4167a0f7407265010077008b689b6926cc13cd2383db30102435df0c75b2dca2989a42e307972c4ec4b9860f38c0684aea69e5b6d6b40a051320de86ac5b3536d8c619f",
+        "wg_public_key": "0VcC6PCOW4JDZERtOUljSKXiPfd0di5qRQ/SHFn9XCs=",
+        "wg_listen_port": 12348,
+        "peers": [
+          {
+            "subnet": "172.20.1.0/24",
+            "wg_public_key": "ZdSn/3gDfD1urtDCOPdOFwF0XsTAidJTSpwPETegZGw=",
+            "allowed_ips": [
+              "172.20.1.0/24",
+              "100.64.20.1/32",
+              "172.20.2.0/24",
+              "100.64.20.2/32"
+            ],
+            "endpoint": "[2a02:1802:5e:0:1000::f6]:12345"
+          },
+          {
+            "subnet": "172.20.3.0/24",
+            "wg_public_key": "tA0tYq2Z+tIeG1oW+1QJ/cJN6FpDX17ogwf3SefaGjs=",
+            "allowed_ips": [
+              "172.20.3.0/24",
+              "100.64.20.3/32"
+            ],
+            "endpoint": "[2a02:1802:5e:0:225:90ff:fef4:2917]:12347"
+          }
+        ]
+      }
+    ]
   },
-  "signature": "",
-  "created": "",
+  "created": "0001-01-01T00:00:00Z",
   "duration": 0,
+  "to_delete": false
 }
 ```
 
+#### Network provisioning graph
```

Comment:
There needs to be a paragraph on the `add-access` command too otherwise user are not able to connect to their network.